code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -313,6 +170,10 @@ var/list/gaslist_cache = init_gaslist_cache()
 	//Compares sample to self to see if within acceptable ranges that group processing may be enabled
 	//Returns: a string indicating what check failed, or "" if check passes
 
+/datum/gas_mixture/proc/react(turf/open/dump_location)

```

Comment:
It's a greater impact than I thought, too. The overhead from defining the first way is 4 times as much as the second way... actually ridiculous.